client-react/src/pages/app/deployment-center/code/DeploymentCenterCodeForm.tsx

```diff
@@ -83,6 +84,23 @@ const DeploymentCenterCodeForm: React.FC<DeploymentCenterCodeFormProps> = props
         properties: payload,
       });
 
+      portalContext.log(getTelemetryInfo('info', 'updateSourceControls', 'submit'));
+
+      if (
+        !!deploymentCenterContext.configMetadata &&
+        !deploymentCenterContext.configMetadata[DeploymentCenterConstants.metadataCurrentStack]
```

Comment:
Disregard if it is deliberate not to run lines 93-101 if `deploymentCenterContext.configMetadata` is undefined.
```suggestion
        !deploymentCenterContext.configMetadata?.[DeploymentCenterConstants.metadataCurrentStack]
```